stylesheets/_component.modals.scss

```diff
@@ -107,8 +107,9 @@
 
 // Title text within header
 .modal__title {
-    margin: 0;
+    font-size: 16px;
```

Comment:
Should this use a font size variable like `$font-size-base` for example?

Reply:
Good point, I totally forgot about it!